### PR TITLE
Replace typeface Circular with Source Sans Pro

### PR DESCRIPTION
**Pull request by StefKors**
Connects-to: #1075 
See: <url> 
Depends-on: <url> 
Change-type: major  

---
##### Contributor checklist

- [ ] I have rebuilt the README with `npm run build:docs`
- [x] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [x] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

File: src/components/Provider/index.tsx [hunk 1]
```diff
@@ -28,6 +29,12 @@ const Provider = ({ theme, ...props }: ThemedProvider) => {
 	const providerTheme = merge(defaultTheme, theme);
 	return (
 		<BreakpointProvider breakpoints={providerTheme.breakpoints}>
+			<Helmet>
+				<link
+					href="https://fonts.googleapis.com/css?family=Nunito|Ubuntu+Mono:400,700|Source+Sans+Pro"
```
Comment (sradevski) [inline src/components/Provider/index.tsx:34]:
```suggestion
					href="https://fonts.googleapis.com/css?family=Nunito|Ubuntu+Mono:400,700|Source+Sans+Pro&display=fallback"
```

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -1,15 +1,6 @@
...
-  const loaded = () => !!document.querySelector('link[loaded="loaded"]');
-  if (loaded()) return Promise.resolve();
-  return new Promise((resolve, reject) => {
-    const id = setInterval(() => {
-      if (!loaded()) return;
-      clearInterval(id);
-      resolve();
-    }, 50);
-  });
+async function fontLoading() {
```
Comment (sradevski) [inline .storybook/preview-head.html:2]: Nice :)

File: src/components/Provider/index.tsx (hunk 1 above)
Comment (sradevski) [reply src/components/Provider/index.tsx:34]: Also, let's see what weights we use from Nunito and Source Sans Pro and only import those. This is something we need to chat with Alexandra, can you please chat with her? BTW the idea is to also replace nunito with source sans pro, but I'd be happy if we do the titles first.

File: src/components/Provider/index.tsx [hunk 3]
```diff
@@ -28,6 +29,12 @@ const Provider = ({ theme, ...props }: ThemedProvider) => {
 	const providerTheme = merge(defaultTheme, theme);
 	return (
 		<BreakpointProvider breakpoints={providerTheme.breakpoints}>
+			<Helmet>
+				<link
+					href="https://fonts.googleapis.com/css?family=Nunito|Ubuntu+Mono:400,700|Source+Sans+Pro&display=fallback"
```
Comment (LucianBuzzo) [inline src/components/Provider/index.tsx:34]: Please add Italic and bold variant

**Comment by StefKors** [comment]
@balena-ci retest

File: src/theme.ts [hunk 4]
```diff
@@ -121,7 +121,7 @@ export const weights = [400, 700];
 export const radius = 3;
 
 export const font = `'Nunito', Arial, sans-serif`;
```
Comment (sradevski) [inline src/theme.ts:123]: Let's make this SSP as well

Comment (StefKors) [reply src/theme.ts:123]: @sradevski should we keep both `font` and `titleFont`?

Comment (sradevski) [reply src/theme.ts:123]: @StefKors yeah let's keep it for now, it doesn't really add that much overhead anyway

Comment (StefKors) [reply src/theme.ts:123]: +1

**Comment by grahammcculloch** [comment]
This has become a bit more urgent as the `circular-std` npm package we were using has been taken down now. Would be good to get this PR merged ASAP 😄

File: src/theme.ts [hunk 5]
```diff
@@ -115,13 +115,16 @@ export const space = [
 	128,
 ];
 
-export const fontSizes = [12, 14, 16, 20, 24, 32, 48, 64, 72, 96];
-export const weights = [400, 700];
+// Old font-scale
+// export const fontSizes = [12, 14, 16, 20, 24, 32, 48, 64, 72, 96];
```
Comment (sradevski) [inline src/theme.ts:119]: We can remove this one

Comment (StefKors) [reply src/theme.ts:119]: Thanks @sradevski I'll go through this and fix them

- [ ] 1. The Badges are very small
- [ ] 2. The Card text is quite large compared to the rest of the components
- [ ] 3. Checkbox labels are quite large (compared to Button text, for example)
- [ ] 4. HighlightedName text is quite small
- [ ] 5. Steps text is quite large, especially when compared to the than the Step title

Comment (StefKors) [reply src/theme.ts:119]: @sradevski `HighlightedName`, is already at the size based on the [design](https://app.zeplin.io/project/5cd17644e6012f68816cca3b/screen/5d62783b38345e9b5a66e786). Shall I make it bigger or do we follow the design size?

Comment (StefKors) [reply src/theme.ts]: @sradevski I think it's ready to go, let me know how it looks in the dashboard 👍

**Pull request merged**